Documentation/contributing.rst

```diff
@@ -181,6 +181,23 @@ If for some reason, running of the provisioning script fails, you should bring t
 
     $ vagrant halt
 
+Packer-CI-Build
```

Comment:
As a developer, when do I care about Packer-CI-Build? How do I know if I should trigger a new build, and how do I trigger it?

Reply:
Done